packages/@react-native-windows/codegen/Document.md

```diff
@@ -0,0 +1,386 @@
+# React Native TurboModule C++ Code Generation
+
+## Running Codegen
+
+`@react-native-windows/codegen` only supports reading turbo modules in Flow today.
+
+`react-native-windows-codegen --file <FILE> --files <FILES> --namespace <NAMESPACE> --libraryName <LIBRARY>`
+
+- `file`: An input file.
+- `files`: a glob pattern for multiple input files.
+- `namespace`: The namespace for the generated spec file. For example: `Microsoft::ReactNativeSpecs`.
+- `libraryName`: part of the output folder for components.
+
+After successfully executed the codegen,
+a C++ spec class will be generated in a header file.
+
+## TypeScript TurboModule Syntax
```

Comment:
>## [](http://example.com/codeflow?start=0&length=2)

The text above says that codegen only supports Flow. Here we talk about TypeScript instead of Flow. Should we change the text above?